tasks/main.yml

```diff
@@ -58,7 +59,16 @@
   when:
     - snapd_purge
   block:
+  - name: Unmount snap
+      become: true
+      mount:
+        path: "/var/snap/firefox/common/host-hunspell"
```

Comment:
Does this only refer to Firefox?
What about other mountpoints?
I would like to see this part in an extra MR and make it more generic.

Reply:
The only problem I had was with Firefox. But I didn't tried this in months and won't be able to do it inn a short amount of time

Reply:
I would accept your MR for the time being and adjust it afterwards so that it meets my requirements.